<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuilderWidget</class>
 <widget class="QFrame">
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <!--property name="styleSheet">
   <string notr="true">QFrame { background-color: palette(window); }</string>
  </property>
  <property name="windowOpacity">
   <number>0.5</number>
  </property-->
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <!--item>
      <widget class="QLabel" name="imageLabel">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item-->
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="centerButtonsHorizontalLayout"/>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="rightButtonsHorizontalLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="builderModeComboBox">
       <item>
        <property name="text">
         <string>Tile</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tile protection</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Subtile</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Subtile protection</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Object</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Monster</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="tileModeWidget">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="firstTileButton">
        <property name="minimumWidth">
         <number>40</number>
        </property>
        <property name="maximumWidth">
         <number>40</number>
        </property>
        <property name="toolTip">
         <string>First tile</string>
        </property>
        <property name="text">
         <string notr="true">|&lt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="previousTileButton">
        <property name="minimumWidth">
         <number>40</number>
        </property>
        <property name="maximumWidth">
         <number>40</number>
        </property>
        <property name="toolTip">
         <string>Previous tile</string>
        </property>
        <property name="text">
         <string notr="true">&lt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="LineEditWidget" name="tileLineEdit">
        <property name="charWidth">
         <number>4</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="nextTileButton">
        <property name="minimumWidth">
         <number>40</number>
        </property>
        <property name="maximumWidth">
         <number>40</number>
        </property>
        <property name="toolTip">
         <string>Next tile</string>
        </property>
        <property name="text">
         <string notr="true">&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="lastTileButton">
        <property name="minimumWidth">
         <number>40</number>
        </property>
        <property name="maximumWidth">
         <number>40</number>
        </property>
        <property name="toolTip">
         <string>Last tile</string>
        </property>
        <property name="text">
         <string notr="true">&gt;|</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="tileProtectionModeWidget">
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Mode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="tileProtectionModeComboBox">
        <item>
         <property name="text">
          <string>Complete Protection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Partial Protection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>No Protection</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="subtileModeWidget">
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="firstSubtileButton">
        <property name="minimumWidth">
         <number>40</number>
        </property>
        <property name="maximumWidth">
         <number>40</number>
        </property>
        <property name="toolTip">
         <string>First subtile</string>
        </property>
        <property name="text">
         <string notr="true">|&lt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="previousSubtileButton">
        <property name="minimumWidth">
         <number>40</number>
        </property>
        <property name="maximumWidth">
         <number>40</number>
        </property>
        <property name="toolTip">
         <string>Previous subtile</string>
        </property>
        <property name="text">
         <string notr="true">&lt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="LineEditWidget" name="subtileLineEdit">
        <property name="charWidth">
         <number>5</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="nextSubtileButton">
        <property name="minimumWidth">
         <number>40</number>
        </property>
        <property name="maximumWidth">
         <number>40</number>
        </property>
        <property name="toolTip">
         <string>Next subtile</string>
        </property>
        <property name="text">
         <string notr="true">&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="lastSubtileButton">
        <property name="minimumWidth">
         <number>40</number>
        </property>
        <property name="maximumWidth">
         <number>40</number>
        </property>
        <property name="toolTip">
         <string>Last subtile</string>
        </property>
        <property name="text">
         <string notr="true">&gt;|</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="subtileProtectionModeWidget">
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Mode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="subtileProtectionModeComboBox">
        <item>
         <property name="text">
          <string>Complete Protection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monster Protection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Object Protection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>No Protection</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="objectModeWidget">
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWidget" name="objectLineEdit">
        <property name="charWidth">
         <number>5</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="objectComboBox">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="monsterModeWidget">
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWidget" name="monsterLineEdit">
        <property name="charWidth">
         <number>5</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="monsterCheckBox"/>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="monsterComboBox">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
